.github/workflows/standard.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       # Extras that will be installed for both conda/pip:
       testing-extras: "happi"
       # Extras to be installed only for conda-based testing:
-      conda-testing-extras: "pcdsdevices pyqt=5"
+      conda-testing-extras: "pyqt=5"
```

Comment:
This can be removed since it's in conda recipe requirements

Reply:
🤦 oh yes